src/css/flags/user-inactive.less

```diff
@@ -11,13 +11,20 @@
           cursor: none;
           -webkit-cursor-visibility: auto-hide;
       }
+      .jw-dock {
+        display: block;
```

Comment:
Thanks for catching that, i didn't realize the '.jwplayer.jw-flag-controls-hidden;' above was triggering controls-hidden.less